networking/kvm.go

```diff
@@ -36,13 +40,18 @@ import (
 	"github.com/coreos/rkt/common"
 	"github.com/coreos/rkt/networking/netinfo"
 	"github.com/coreos/rkt/networking/tuntap"
+	"github.com/coreos/rkt/pkg/lock"
 )
 
 const (
-	defaultBrName     = "kvm-cni0"
-	defaultSubnetFile = "/run/flannel/subnet.env"
-	defaultMTU        = 1500
-	masqComment       = "rkt-lkvm masquerading"
+	defaultBrName         = "kvm-cni0"
+	defaultSubnetFile     = "/run/flannel/subnet.env"
+	defaultMTU            = 1500
+	masqComment           = "rkt-lkvm masquerading"
+	kvmSettingsDir        = "/var/lib/rkt-stage1-kvm"
```

Comment:
Hmmm... I didn't know we had this hardcoded :scream:

We should probably make it part of `--local_config` or `--dir` (not sure which one is best) although we don't have a way to access those from here now.

 /cc @krnowak